tools/scalar-schema/README.md

```diff
@@ -103,3 +102,14 @@ $ java -jar target/scalar-schema-standalone-<version>.jar --help
 ```
 - `compaction-strategy` should be `STCS`, `LCS` or `TWCS`. This is ignored in Cosmos DB and DynamoDB.
 - This `ru` value is set for all tables on this database even if `-r BASE_RESOURCE_UNIT` is set when Cosmos DB and DynamoDB. `ru` is ignored in Cassandra.
+
+## Performance tuning
```

Comment:
I updated the descriptions based on the original ones. 
(First, I updated one by one but it got messy so I created a new doc)
It might be wrong so please check 🙇 
```
## Scaling Performance

### RU

You can scale the throughput of Cosmos DB and DynamoDB by specifying `-r` option (which applies to all the tables) or `ru` parameter for each table. Those configurations are ignored in Cassandra. The default values are `400` for Cosmos DB and `10` for DynamoDB respectively, which are set with `-r` option.

Note that the schema tool abstracts [Request Unit](https://docs.microsoft.com/azure/cosmos-db/request-units) of Cosmos DB and [Capacity Unit](https://docs.aws.amazon.com/amazondynamodb/latest/developerguide/HowItWorks.ReadWriteCapacityMode.html#HowItWorks.ProvisionedThroughput.Manual) of DynamoDB with `RU`.
So, please set an appropriate value depending on the database implementations. Please also note that the schema tool sets the same value to both Read Capacity Unit and Write Capacity Unit for DynamoDB.

### Auto-scaling

By default, the schema tool enables auto-scaling of RU for all tables:  RU is scaled in or out between 10% and 100% of a specified RU depending on a workload. For example, if you specify `-r 10000`, RU of each table is scaled in or out between 1000 and 10000.  Note that auto-scaling of Cosmos DB is enabled only when you set more than or equal to 4000 RU.
```

Questions.
1. The default values are specified with `-r` implicitly? (so, for example, `400` is applied to each table in Cosmos DB?)

Reply:
Thank you for the suggestion!
I've replaced it with your suggestion.

> The default values are specified with -r implicitly? (so, for example, 400 is applied to each table in Cosmos DB?)
The default values are set implicitly when you don't add `-r` option.